pallets/funding/src/lib.rs

```diff
@@ -18,96 +18,51 @@
-//! | `start_auction`                               | X      |                 |                       |                        |
-//! | `bond_evaluation`                             |        | X               | X                     | X                      |
-//! | `failed_evaluation_unbond_for`                |        | X               | X                     | X                      |
-//! | `bid`                                         |        |                 | X                     | X                      |
-//! | `contribute`                                  |        | X               | X*                    | X*                     |
-//! | `vested_plmc_bid_unbond_for`                  |        |                 | X                     | X                      |
-//! | `vested_plmc_purchase_unbond_for`             |        | X               | X                     | X                      |
-//! | `vested_contribution_token_bid_mint_for`      |        |                 | X                     | X                      |
-//! | `vested_contribution_token_purchase_mint_for` |        | X               | X                     | X                      |
-//!
-//! _* They can call contribute only if the project is on the remainder round._
-//!
+//! 1) **Project Creation**: Issuer creates a project with the [`create_project`](Pallet::create_project) extrinsic.
+//! 2) **Evaluation Start**: Issuer starts the evaluation round with the [`start_evaluation`](Pallet::start_evaluation) extrinsic.
+//! 3) **Evaluate**: Evaluators bond PLMC on the project with the [`bond_evaluation`](Pallet::evaluate) extrinsic.
+//! 4) **Evaluation End**: Anyone can end the evaluation round with the [`end_evaluation`](Pallet::end_evaluation) extrinsic after the defined end block.
+//! 5) **Auction Start**: If the project got >=10% of its requested funding (USD) in PLMC bonded, the auction starts immediately after `evaluation_end` was called.
+//! 6) **Bid**: Professional and Institutional investors can bid on the project with the [`bid`](Pallet::bid) extrinsic. Price starts at the issuer defined minimum price, and goes up in a price delta and bucket size of 10%
+//! 7) **Auction End**: Anyone can end the auction round with the [`end_auction`](Pallet::end_auction) extrinsic after the defined end block.
+//! 8) **Community Round Start**: After `end_auction` is called, a weighted average price is calculated from the bids, and the community round starts.
+//! 9) **Contribute**: Anyone without a winning bid can now buy tokens at the calculated price with the [`contribute`](Pallet::contribute) extrinsic.
+//! 10) **Remainder Round Start**: After a defined [period](<T as Config>::CommunityRoundDuration) , the remainder round starts.
+//! 11) **Contribute**: Participants with winning bids can also now buy tokens at the calculated price with the [`contribute`](Pallet::contribute) extrinsic.
```

Comment:
**Contribute**: Participants with winning bids can also contribute at the weighted average price with the [`contribute`](Pallet::contribute) extrinsic.